tommath.h

```diff
@@ -102,8 +102,10 @@ extern "C" {
 /* use arc4random on platforms that support it */
 #ifdef MP_USE_ALT_RAND
     #define MP_GEN_RANDOM()    arc4random()
+    #define MP_GEN_RANDOM_MAX  0xffffffff
```

Comment:
I'm going on a limb here... You should increase `current` if a change breaks the ABI and change the behavior. Here, a random number is generated in a different way, but I don't see this change done in an incompatible way. 

As rakudo packager for Debian, I'll make sure that rakudo uses the new version of libtommath at runtime to avoid the bug mentioned by @timo .

All the best